src/Firebase/Factory.php

```diff
@@ -245,6 +257,39 @@ public function withClock(object $clock): self
         return $factory;
     }
 
+    public function createAppCheck(): Contract\AppCheck
+    {
+        $projectId = $this->getProjectId();
+        $serviceAccount = $this->getServiceAccount();
+
+        if ($projectId === '') {
+            throw new RuntimeException('Unable to determine the Firebase Project ID');
+        }
+
+        if ($serviceAccount === null) {
+            throw new RuntimeException('Unable to use AppCheck without credentials');
+        }
+
+        $http = $this->createApiClient([
+            'base_uri' => 'https://firebaseappcheck.googleapis.com/v1/projects/'.$projectId,
+        ]);
+
+        $keySet = new CachedKeySet(
+            'https://firebase.google.com/docs/reference/appcheck/rest/v1/jwks#resource:-publicjwkset',
```

Comment:
Why is this a link to the Firebase documentation? 😅 

```suggestion
            'https://firebaseappcheck.googleapis.com/v1/jwks',
```